### ISSUE TITLE
Error `undefined method clear_active_connections!`

### ISSUE THREAD

**Issue by dvogtfstr** [plashchynski/crono#128]
hey ✌ getting following error, someone an idea?

```ruby
[2024-12-13T14:23:41.064525 #70777] ERROR -- : [...]_job.rb:7:in `perform'
.../3.1.6/lib/ruby/gems/3.1.0/gems/crono-2.0.1/lib/crono/job.rb:80:in `perform_job'
.../3.1.6/lib/ruby/gems/3.1.0/gems/crono-2.0.1/lib/crono/job.rb:43:in `block in perform'
#<Thread:0x000000010ace79e8 .../3.1.6/lib/ruby/gems/3.1.0/gems/crono-2.0.1/lib/crono/job.rb:43 run> terminated with exception (report_on_exception is true):
.../3.1.6/lib/ruby/gems/3.1.0/gems/activerecord-7.2.2.1/lib/active_record/dynamic_matchers.rb:22:in `method_missing': undefined method `clear_active_connections!' for ActiveRecord::Base:Class (NoMethodError)
Did you mean?  clear_validators!
        from .../3.1.6/lib/ruby/gems/3.1.0/gems/crono-2.0.1/lib/crono/job.rb:50:in `block in save'
        from .../3.1.6/lib/ruby/gems/3.1.0/gems/crono-2.0.1/lib/crono/job.rb:47:in `synchronize'
        from .../3.1.6/lib/ruby/gems/3.1.0/gems/crono-2.0.1/lib/crono/job.rb:47:in `save'
        from .../3.1.6/lib/ruby/gems/3.1.0/gems/crono-2.0.1/lib/crono/job.rb:89:in `perform_job'
        from .../3.1.6/lib/ruby/gems/3.1.0/gems/crono-2.0.1/lib/crono/job.rb:43:in `block in perform'
undefined method `clear_active_connections!' for ActiveRecord::Base:Class
Did you mean?  clear_validators!

```
Something wrong [here](https://github.com/plashchynski/crono/blob/2e663c210415dac584b7eaede27657161a8ccdd6/lib/crono/job.rb#L50)

thanks :)

**Comment by dvogtfstr** [comment]
@plashchynski could you create a new release with [this](https://github.com/plashchynski/crono/pull/127) inlcuded 🙏🏾  please

**Comment by plashchynski** [comment]
Hey @dvogtfstr published v2.1.0 with this change. Thank you
